fastapi_auth_api_key/dependencies.py

```diff
@@ -14,31 +14,27 @@
 
 from fastapi import Depends, status
 from fastapi.exceptions import HTTPException
-from fastapi.security import APIKeyHeader
-
-HTTP_API_KEY_HEADER = os.environ.get("FASTAPI_AUTH_HTTP_API_KEY_HEADER", "HTTP-API-KEY")
 
 
 def authenticated_auth_api_key(
-    key: Annotated[str, Depends(APIKeyHeader(name=HTTP_API_KEY_HEADER))],
     env: Annotated[Environment, Depends(odoo_env)],
     endpoint: Annotated[FastapiEndpoint, Depends(fastapi_endpoint)],
 ) -> AuthApiKey:
+    admin_env = Environment(env.cr, SUPERUSER_ID, {})
+    key = endpoint.sudo().auth_http_api_key_header
     if not key:
         raise HTTPException(
             status_code=status.HTTP_401_UNAUTHORIZED,
-            detail=_("Missing %(HTTP_API_KEY_HEADER)s header")
-            % {"HTTP_API_KEY_HEADER": HTTP_API_KEY_HEADER},
-            headers={"WWW-Authenticate": HTTP_API_KEY_HEADER},
+            detail=admin_env._("Missing HTTP API key header"),
```

Comment:
This is also not correct.

it should report the header to use, kind of like before